spec/models/recommend_room_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+describe RecommendRoom do
+  context "with correct param" do
+    it "is valid" do
+      recommend = create(:recommend_room)
+      expect(recommend).to be_valid
+    end
+  end
+
+  context "with duplicated room" do
+    let(:room) { create(:public_room).tap {|room| create(:recommend_room, room: room) } }
```

Comment:
__[RuboCop]__ Layout/SpaceInsideBlockBraces: Space between { and | missing. ([link](https://sider.review/gh/repos/105680869/pulls/106#issue-3979013))

<sub>posted by Sider</sub>